src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3466,43 +3464,24 @@ export class StudyViewPageStore {
         const cancerTypeIds = _.uniq(
             this.queriedPhysicalStudies.result.map(study => study.cancerTypeId)
         );
+        const survivalUniqueKeys = this.survivalPlotKeys;
```

Comment:
looks like `this.survivalPlotKeys` can be empty here this.survivalClinicalAttributesPrefix.result is not complete. I think `this.survivalClinicalAttributesPrefix.pending` should be added to `loadingInitialDataForSummaryTab`

@leexgh @alisman @zhx828